mne_bids/io.py

```diff
@@ -83,3 +92,72 @@ def _read_raw(raw_fname, electrode=None, hsp=None, hpi=None, config=None,
         raise ValueError('Raw file name extension must be one of {}\n'
                          'Got {}'.format(ALLOWED_EXTENSIONS, ext))
     return raw
+
+
+def read_raw_bids(bids_fname, output_path, return_events=True,
+                  verbose=True):
+    """Read BIDS compatible data.
+
+    Parameters
+    ----------
+    bids_fname : str
+        Full name of the data file
+    output_path : str
```

Comment:
I find `output_path` a strange name of the parameter. It makes only sense if a user has applied some write function beforehand ... if a user is "only" reading, the parameter name loses its meaning.

I suggest `bids_dir` or `bidsdir_path`